pbcommand/models/report.py

```diff
@@ -85,7 +85,7 @@ def __init__(self, id_):
             raise PbReportError("Type error. id '{i}' cannot be {t}.".format(i=id_, t=type(id_)))
 
         if not re.match('^[a-z0-9_]+$', id_):
```

Comment:
I strongly recommend *never* using `re.match()`, since it is asymmetrically anchored. Always prefer `re.search()`.

In this case, they're the same because of the `$`, but it's a good habit.